lib/mix/lib/mix/local/installer.ex

```diff
@@ -213,7 +213,12 @@ defmodule Mix.Local.Installer do
         package_name
       end
 
-    {:fetcher, {String.to_atom(app_name), version, hex: String.to_atom(package_name)}}
+    dep_opts =
+      opts
+      |> Keyword.take([:organization])
+      |> Keyword.merge(hex: String.to_atom(package_name))
```

Comment:
Can we use Keyword.put here instead?

Reply:
Yes, this was brought over from moving stuff around. Pushed just now.